app/views/content_items/_content_item.html.erb

```diff
@@ -7,5 +7,9 @@
   <% end %>
   <td><%= content_item.document_type %></td>
   <td><%= content_item.unique_page_views %></td>
-  <td><%= time_ago_in_words(content_item.public_updated_at) %> ago</td>
+  <td>
+    <% if content_item.public_updated_at %>
+    <%= time_ago_in_words(content_item.public_updated_at) %> ago
+    <% end %>
```

Comment:
Some default text e.g never might be a good idea here, so we don't leave empty table cells.